core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -162,14 +162,11 @@ oppia.directive('answerGroupEditor', [
             var ruleTypes = Object.keys(ruleDescriptions);
             var ruleType = null;
             for (var i = 0; i < ruleTypes.length; i++) {
-              if (ruleTypes[i] !== RULE_TYPE_CLASSIFIER) {
-                ruleType = ruleTypes[i];
-                break;
-              }
+              ruleType = ruleTypes[i];
```

Comment:
I'm confused. What's going on here? It seems like you just keep overwriting ruleType; the original code had a 'break'.

I think this can be simplified a lot.

Reply:
Yes, I forgot this piece of code. Will do in next commit.

Reply:
Done